requests_pkcs12.py

```diff
@@ -27,10 +26,9 @@
 import requests.adapters
 import ssl
 import tempfile
-import urllib3.contrib.pyopenssl
 
 try:
-    from ssl import PROTOCOL_TLS as default_ssl_protocol
+    from ssl import PROTOCOL_TLS_CLIENT as default_ssl_protocol
```

Comment:
This fixes another deprecation warning as in [ssl docs](https://docs.python.org/3/library/ssl.html#ssl.PROTOCOL_TLS).